modules/user_accounts/php/edit_user.class.inc

```diff
@@ -973,7 +973,21 @@ class Edit_User extends \NDB_Form
 
                 if (strlen($effectiveUID) > 255) {
                     $errors['UserID_Group']
-                        = 'The user name must not exsceed 255 characters';
+                        = 'The user name must not exceed 255 characters';
+                }
+
+                // Check that user name does not contain a whitespace character
+                if (preg_match('/\s/', $effectiveUID)) {
+                    // Note: email adresses can contain comments which themselves can
+                    // contain spaces
```

Comment:
That's true for mail clients software, but is it true for LORIS?

The name part of `"John Smith" <john.smith@example.com>` is constructed, not user supplied. (Assuming that's what you're referring to..)

Reply:
You can enter an email address like `me(yeah really me)@test.com` on the create new user page in LORIS, and that is a valid email address with a user-supplied comment. My PR prevents the user name from being set to the email address in such a (rare/extreme) case.

Reply:
TIL. (That's an insane thing for the spec to allow.)